kadena-signing-api/src/Kadena/SigningTypes.hs

```diff
@@ -0,0 +1,147 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Kadena.SigningTypes where
+
+import Control.Lens hiding ((.=))
+import Control.Monad
+import Data.Aeson
+import qualified Data.Aeson as A
+import Data.Aeson.Types
+import qualified Data.Char as Char
+import qualified Data.HashMap.Strict as HM
+import qualified Data.List.Split as L
+import qualified Data.Map as M
+import Data.Maybe
+import Data.Text (Text)
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
+import GHC.Generics
+
+import Pact.Parse
+import Pact.Types.Command
+import Pact.Types.Hash
+-- TODO: Rip out sig data dependency
+import Pact.Types.SigData
+
+newtype SignatureList = SignatureList  [(PublicKeyHex, Maybe UserSig)]
```

Comment:
While we're here can you add an accessor `unSignatureList` or similar so that the user doesn't have to pattern match?